apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.ts

```diff
@@ -63,15 +66,52 @@ router.get(
 router.get(
   '/',
   asyncHandler(async (req, res) => {
+    const manualGradingPreviewEnabled = req.query.manual_grading_preview === 'true';
+    if (manualGradingPreviewEnabled) {
+      // Setting this flag will set the `manualGrading: true` flag in the data
+      // dictionary passed to element render functions. It will also disable
+      // editing for all elements by settings `editable: false`.
+      res.locals.manualGradingInterface = true;
+    }
+
     const variant_seed = req.query.variant_seed ? z.string().parse(req.query.variant_seed) : null;
     const variant_id = req.query.variant_id ? IdSchema.parse(req.query.variant_id) : null;
     // req.query.variant_id might be undefined, which will generate a new variant
     await getAndRenderVariant(variant_id, variant_seed, res.locals as any);
     await logPageView('instructorQuestionPreview', req, res);
     await setQuestionCopyTargets(res);
 
+    const searchParams = getSearchParams(req);
+
+    // Construct a URL to preview the question as it would appear in the manual
+    // grading interface. We need to include the `variant_id` in the URL so that
+    // we show the current variant and not a new one.
+    const manualGradingPreviewSearchParams = new URLSearchParams(searchParams);
+    manualGradingPreviewSearchParams.set('variant_id', res.locals.variant.id.toString());
+    manualGradingPreviewSearchParams.set('manual_grading_preview', 'true');
```

Comment:
Nit: do you want to delete `variant_seed`? It won't make a difference in behaviour since this is only used when `variant_id` is not set, but it's probably a good idea not to persist a param that isn't actually used. I won't block on it. Same for the normal preview below.

Reply:
Done in 382e524.